mysql/README.md

```diff
@@ -49,29 +51,37 @@ grep Uptime && echo -e "\033[0;32mMySQL user - OK\033[0m" || \
 echo -e "\033[0;31mCannot connect to MySQL\033[0m"
 ```
 
-```shell
-mysql -u datadog --password=<UNIQUEPASSWORD> -e "show slave status" && \
-echo -e "\033[0;32mMySQL grant - OK\033[0m" || \
-echo -e "\033[0;31mMissing REPLICATION CLIENT grant\033[0m"
-```
+The Agent needs a few privileges to collect metrics. Grant the `datadog` user only the following limited privileges.
 
-The Agent needs a few privileges to collect metrics. Grant the user the following limited privileges ONLY:
+For MySQL versions 5.6 and 5.7, set `max_user_connections` with the following command:
 
 ```shell
 mysql> GRANT REPLICATION CLIENT ON *.* TO 'datadog'@'%' WITH MAX_USER_CONNECTIONS 5;
 Query OK, 0 rows affected, 1 warning (0.00 sec)
+```
 
-mysql> GRANT PROCESS ON *.* TO 'datadog'@'%';
+For MySQL 8.0 or greater, set `max_user_connections` with the following command:
+
+```shell
+mysql> ALTER USER 'datadog'@'%' WITH MAX_USER_CONNECTIONS 5;
 Query OK, 0 rows affected (0.00 sec)
 ```
 
-For MySQL 8.0+ set `max_user_connections` with:
+Grant the `datadog` user the process privilege:
 
 ```shell
-mysql> ALTER USER 'datadog'@'%' WITH MAX_USER_CONNECTIONS 5;
+mysql> GRANT PROCESS ON *.* TO 'datadog'@'%';
```

Comment:
I had a related docs ticket in my to do list (DOCS-3704) that said this space was missing a step. Does your change also remove the need to add the step of granting access to tables? Such as:

`GRANT SELECT ON <YOUR_SCHEMA>.<tables> TO 'datadog'@'%'`

or

`GRANT SHOW VIEW ON <YOUR_SCHEMA>.<tables> TO 'datadog'@'%'`